viite-backend/viite-main/src/main/scala/fi/liikennevirasto/viite/ProjectService.scala

```diff
@@ -1662,8 +1678,8 @@ class ProjectService(
             calculatedLinks
       }.toSeq
 
-      val terminatedProjectLinksWithAssignedRoadwayNumbers = assignRoadwayNumbersToTerminatedProjectLinks(projectLinks)
-      val originalAddresses = roadAddressService.getRoadAddressesByRoadwayIds((recalculated ++ terminated).map(_.roadwayId))
+      val terminatedProjectLinksWithAssignedRoadwayNumbers = assignRoadwayNumbersToTerminatedProjectLinks(others ++ terminatedProjectLinks)
```

Comment:
Nimeä uudelleen näitä recalculated, others, jne. (Sulla ilmeisesti oli jo rafaktorointeja, jotka hävisivät mergen myötä)